test/extended/etcd/etcd_storage_path.go

```diff
@@ -224,16 +224,19 @@ func testEtcd3StoragePath(t g.GinkgoTInterface, kubeConfig *restclient.Config, e
 	defer ht.done()
 	t = ht
 
-	var tt *testing.T // will cause nil panics that make it easy enough to find where things went wrong
-
 	kubeConfig = restclient.CopyConfig(kubeConfig)
 	kubeConfig.QPS = 99999
 	kubeConfig.Burst = 9999
 	kubeClient := kclientset.NewForConfigOrDie(kubeConfig)
 	crdClient := apiextensionsclientset.NewForConfigOrDie(kubeConfig)
 
 	// create CRDs so we can make sure that custom resources do not get lost
-	etcddata.CreateTestCRDs(tt, crdClient, false, etcddata.GetCustomResourceDefinitionData()...)
+	for _, crd := range etcddata.GetCustomResourceDefinitionData() {
+		if _, err := crdClient.ApiextensionsV1beta1().CustomResourceDefinitions().Create(crd); err != nil {
+			t.Fatalf("Failed to create %s CRD; %v", crd.Name, err)
```

Comment:
now it prints `fail [github.com/onsi/ginkgo/internal/leafnodes/runner.go:109]: Failed to create pants.custom.fancy.com CRD; customresourcedefinitions.apiextensions.k8s.io "pants.custom.fancy.com" already exists`

instead of 
```
  /usr/local/Cellar/go/1.12.9/libexec/src/runtime/panic.go:522 +0x1b5
github.com/openshift/origin/vendor/k8s.io/kubernetes/test/integration/etcd.createTestCRD(0x0, 0x691c7c0, 0xc002e536d0, 0xc00178d000, 0xc00178cb00)
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/test/integration/etcd/server.go:305 +0x384
github.com/openshift/origin/vendor/k8s.io/kubernetes/test/integration/etcd.CreateTestCRDs(0x0, 0x691c7c0, 0xc002e536d0, 0x6a64500, 0xc003c04120, 0x3, 0x3)
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/test/integration/etcd/server.go:299 +0x61
github.com/openshift/origin/test/extended/etcd.testEtcd3StoragePath(0x6a38020, 0xc002f896c0, 0xc00000d0e0, 0x69cd320, 0xc002f87200)
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/etcd/etcd_storage_path.go:236 +0x188
github.com/openshift/origin/test/extended/etcd.glob..func1.1()
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/etcd/etcd_test_runner.go:72 +0x12d6
github.com/openshift/origin/vendor/github.com/onsi/ginkgo/internal/leafnodes.(*runner).runSync(0xc0034127e0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/cmd/openshift-tests/openshift-tests.go:234 +0x11f
github.com/openshift/origin/vendor/github.com/spf13/cobra.(*Command).execute(0xc002c56780, 0xc003543f30, 0x1, 0x1, 0xc002c56780, 0xc003543f30)
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/github.com/spf13/cobra/command.go:756 +0x465
github.com/openshift/origin/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc002c56000, 0x68d7378, 0xa6070f0, 0xa6070f0)
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/github.com/spf13/cobra/command.go:846 +0x2ec
github.com/openshift/origin/vendor/github.com/spf13/cobra.(*Command).Execute(...)
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/github.com/spf13/cobra/command.go:794
main.main.func1(0xc002c56000, 0x0, 0x0)
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/cmd/openshift-tests/openshift-tests.go:71 +0x93
main.main()
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/cmd/openshift-tests/openshift-tests.go:72 +0x327

fail [runtime/panic.go:522]: Test Panicked: runtime error: invalid memory address or nil pointer dereference

failed: (13.1s) 2019-09-16T11:23:42 "[Serial] API data in etcd should be stored at the correct location and version for all resources [Suite:openshift/conformance/serial]"

Failing tests:
[Serial] API data in etcd should be stored at the correct location and version for all resources [Suite:openshift/conformance/serial]

error: 1 fail, 0 pass, 0 skip (13.1s)
```